sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithStateSuite.scala

```diff
@@ -1830,6 +1831,19 @@ abstract class TransformWithStateSuite extends StateStoreMetricsTest
           CheckNewAnswer(("a", "1")),
           StopStream
         )
+
+        val hadoopConf = spark.sessionState.newHadoopConf()
```

Comment:
I understand this is to check whether we filter the files which aren't bound to offset log / schema log / etc. Shall we leave a comment for this? Also, shall we check whether we do not filter them out during pruning?